.github/workflows/boolean_benchmark.yml

```diff
@@ -118,7 +102,7 @@ jobs:
       - name: Send data to Slab
         shell: bash
         env:
-          COMPRESSED_RESULTS : ${{ env.RESULTS_FILENAME }}.gz
+          COMPRESSED_RESULTS: ${{ env.RESULTS_FILENAME }}.gz
```

Comment:
All occurrences of COMPRESSED_RESULTS can be removed throughout all the bench files.
It's an artefact from one of my devs.

Reply:
Ok will do.